src/Security.php

```diff
@@ -55,4 +55,21 @@ public static function wp_redirect($url) {
     return $url;
   }
 
+  /**
+   * @implemements wp_headers
+   *
+   * @return array
+   */
+  public static function additional_securityheaders(array $headers):array {
```

Comment:
1. Minor: Missing space before return type hint.
    ```suggestion
      public static function additional_securityheaders(array $headers): array {
    ```

2. Minor: Missing blank line after function declaration (before closing class).